src/main/java/school/faang/user_service/controller/event/EventParticipationController.java

```diff
@@ -0,0 +1,55 @@
+package school.faang.user_service.controller.event;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import school.faang.user_service.dto.UserDto;
+import school.faang.user_service.entity.User;
+import school.faang.user_service.service.event.EventParticipationService;
+import school.faang.user_service.validator.event.EventParticipationValidator;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/events")
+public class EventParticipationController {
+
+    private final EventParticipationService eventParticipationService;
+    private final EventParticipationValidator participationValidator;
+
+    @PostMapping("/{eventId}/{userId}")
+    public ResponseEntity<String> registerParticipant(@PathVariable long eventId, @PathVariable long userId){
+        participationValidator.checkForNull(eventId, userId);
+        eventParticipationService.registerParticipant( eventId, userId );
+        return ResponseEntity.ok("User " + userId + " registered");
```

Comment:
In methods further you are returning ResponseEntity<> in service, so you to unify your code design it is better to choose one of the approaches.